draft-ietf-rtcweb-jsep.xml

```diff
@@ -2822,7 +2829,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           section MUST also be generated for data. The &lt;media&gt;
           field MUST be set to "application" and the &lt;proto&gt; and
           "fmt" fields MUST be set to exactly match the fields in the
-          offer.</t>
+          offer. Note that if media m= sections are bundled into a
+          data m= section, then the TRANSPORT and IDENTICAL attributes
```

Comment:
As not all attributes are relevant here, suggest that you replace 'the' with 'certain'